Python Program to Sort Words in Alphabetic Order.py

```diff
@@ -1,18 +1,18 @@
-# Program to sort alphabetically the words form a string provided by the user
+# Program to sort words alphabetically and print them out in a list
 
+#declaring variables
 my_str = "Hello this Is an Example With cased letters"
+word_Dict = {}
+count = 0
```

Comment:
Why this variable **count** is used for no reason of it being anywhere in the program.